lib/telesign/api.rb

```diff
@@ -43,6 +43,8 @@ def initialize opts = {}
         end
         faraday.adapter  Faraday.default_adapter  # make requests with Net::HTTP
       end
+      @conn.request.options.timeout = opts[:timeout] if opts[:timeout].present?
```

Comment:
`#present?` is a rails/active support method not a standard thing in ruby, this gem does not require rails so it would fail anywhere else but in a rails environment.